Automattic/jetpack
# My Jetpack: add feature as possible recommendations

## Proposed changes:

* This PR updates the frontend of My Jetpack in order to display feature modules as possible recommendations. The product cards were adapted to handle feature and their various statuses.

NOTE: the new feature cards work similar to how the product cards work. There's an issue where the state isn't immediately reflected when we, for example, activate a module. That behavior should be fixed in a follow-up.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch and enable sandbox access
* Go to Jetpack Constants and update `JETPACK__SANDBOX_DOMAIN` to your sandbox domain
* Checkout this branch in your sandbox: 168603-ghe-Automattic/wpcom
* Connect My Jetpack, and wait until you see the recommendations survey
* Check "Grow my site" and wait until you see the recommendations
* Confirm that you can see and interact with "Related Posts" and "Newsletter"
